magento/magento2
# Add 'schedule status' column to admin indexer grid

### Description (*)

This copies the 'schedule status' column from command line (`php bin/magento indexer:status`) into the admin grid for indexes.

### Fixed Issues (if relevant)


### Manual testing scenarios (*)

1. Stop/disable Magento cron.
1. Set indexes to 'update on schedule'.
1. Note column in admin grid: system -> tools -> index management
1. Change product details which will need to be indexed.
1. Observe column in grid
1. Start/re-enable Magento cron.
1. Observe column in grid

Also compare admin grid content to command line content at relevant steps.

### Questions or comments

I do not know if this requires a unit test.

### Contribution checklist (*)
